google-cloud-function/main.py

```diff
@@ -0,0 +1,75 @@
+import datetime
+import json
+import sys
+
+# from flask import jsonify
```

Comment:
I would remove this

Reply:
I had intended to, I forgot it was in there

done